src/Resources/app/administration/src/module/sw-cms/elements/twitter-timeline/config/sw-cms-el-config-twitter-timeline.html.twig

```diff
@@ -1,11 +1,43 @@
 {% block sw_cms_element_twitter_timeline_config %}
-    <div class="sw-cms-el-config-twitter-timeline">
+<div class="sw-cms-el-config-twitter-timeline">
+    {% block sw_cms_element_twitter_timeline_config_twitter_handle %}
         <sw-field class="twitter-handle-field"
                   v-model="element.config.twitterHandle.value"
                   type="text"
-                  label="Twitter Timeline"
-                  placeholder="Enter Twitter Handle."
-                  @blur="onBlur">
+                  :label="$tc('sw-cms.elements.twitterHandle.label')"
+                  :placeholder="$tc('sw-cms.elements.twitterHandle.placeHolder')">
         </sw-field>
-    </div>
+    {% endblock %}
+
+    {% block sw_cms_element_twitter_timeline_config_timeline_type %}
+    <sw-select-field v-model="element.config.timelineType.value"
+                     :label="$tc('sw-cms.elements.twitterTimeline.type.label')"
+                     :placeholder="$tc('sw-cms.elements.twitterTimeline.type.placeHolder')">
+        <option value="profile">{{ $tc('sw-cms.elements.twitterTimeline.type.profile') }}</option>
```

Comment:
Options can be put inside `src/Resources/app/administration/src/module/sw-cms/elements/twitter-timeline/config/index.js`

like this:
```javascript
data() {
        return {
            listTypes: [
                {
                    id: 'option1',
                    name: "label1",
                },
                {
                    id: 'option2',
                    name: "label2",
                },
            ],
        };
    },
```

Reply:
This is intressting. Funny in SW6 core they also add options in template.

Reply:
Hmm I was think also about this, but then I have to add more code just for some option that will not change that often. So I would prefer to leave it small and everything in the template.

You need to do something like this:
https://vuejs.org/v2/guide/forms.html#Select